### PR TITLE
add Oum language

### PR DESCRIPTION
**Pull request by Theodus**
addition of the oum.html.markdown file

### CODE REVIEW

File: oum.html.markdown [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+// division of integers returns an integer
+
+5 / 2  //=> 2
+-5 / 2 //=> -2
+
+// if at least one of the nubers is a double value a double will be returned
+
+5.0 / 2 //=> 2.5
+
+// modulus operation returns the remainder of a division
```
Comment (vendethiel) [inline oum.html.markdown]: you're sometimes using "exponent" and sometimes "modulus **operation**". any reason?

**Comment by vendethiel** [comment]
:+1:

**Comment by geoffliu** [comment]
What is the current usage of this language? From the github page, it seems like it's fairly new and untested. A quick Google search yielded no homepage, no documentation, nor any mentions on any other site. I don't think it's suitable for LXIY at this time. What do other maintainers think?

**Comment by vendethiel** [comment]
I don't feel **strongly** otherwise, I just think it doesn't cost much to have a tutorial for it.

**Pull request closed without merging**